src/Symfony/Component/Intl/Resources/data/locales/tt.json

```diff
@@ -508,19 +508,19 @@
         "yo_BJ": "йоруба (Бенин)",
         "yo_NG": "йоруба (Нигерия)",
         "zh": "кытай (тәрҗемә киңәше: аерым алганда, мандарин кытайчасы)",
-        "zh_CN": "кытай (Кытай)",
-        "zh_HK": "кытай (Гонконг Махсус Идарәле Төбәге)",
-        "zh_Hans": "кытай (гадиләштерелгән)",
-        "zh_Hans_CN": "кытай (гадиләштерелгән, Кытай)",
-        "zh_Hans_HK": "кытай (гадиләштерелгән, Гонконг Махсус Идарәле Төбәге)",
-        "zh_Hans_MO": "кытай (гадиләштерелгән, Макао Махсус Идарәле Төбәге)",
-        "zh_Hans_SG": "кытай (гадиләштерелгән, Сингапур)",
-        "zh_Hant": "кытай (традицион)",
-        "zh_Hant_HK": "кытай (традицион, Гонконг Махсус Идарәле Төбәге)",
-        "zh_Hant_MO": "кытай (традицион, Макао Махсус Идарәле Төбәге)",
-        "zh_Hant_TW": "кытай (традицион, Тайвань)",
-        "zh_MO": "кытай (Макао Махсус Идарәле Төбәге)",
-        "zh_SG": "кытай (Сингапур)",
-        "zh_TW": "кытай (Тайвань)"
+        "zh_CN": "кытай (тәрҗемә киңәше: аерым алганда, мандарин кытайчасы) (Кытай)",
```

Comment:
this looks scary, but seems the legit translation of `zh` primary language :| (see https://github.com/unicode-org/icu/blob/master/icu4c/source/data/lang/tt.txt)

playing a bit with google translate i get `Chinese (Interpretation of Interpretation: Mandarin Chinese)`, but im not sure if anyone speaks `tt` (Tatar?)

Reply:
as we already used this translation for `zh` directly (see above), im not sure we should care for now.

From here on we should rely on native speakers to add a possible "per case fix" IMHO.